backend/demo_consolidated_memory.py

```
@@ -0,0 +1,161 @@
+    ]
+    
+    # Save fragmented memories
+    for memory in fragmented_memories:
+        file_service.save_memory(project_id, memory)
+    
+    print(f"   📝 Created {len(fragmented_memories)} fragmented memories")
+    
+    # Perform migration
+    migration_results = service.migrate_to_consolidated_memories(project_id)
+    
+    print(f"   🔄 Migration results:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)